### ISSUE TITLE
Missing aria options for accesibility in TooltipHost Component

### ISSUE THREAD

**Issue by Nianwen** [OfficeDev/office-ui-fabric-react#1299]
In VSTS, we are adopting the TooltipHost across all areas, but to set it as good pattern, it missing the ability to set aria options to get better accessibility experience.

**Comment by mdahamiwal** [comment]
@nianwen, what are the exact aria options you are looking for or seem missing? Tooltips in general are not focusable elements and we already attach `aria-describedby` for the wrapping element.

**Comment by Nianwen** [comment]
@mdahamiwal , my initial thought was, we could give user the flexibility to user to specify some options, such as aria-label or aria-labelledby, but as for now aria-describedby may works, do you mind point me the place, I did not find it my DOM. I also see some people report the accessible issue on screen reader as in issue #1301 . Known issue?

**Comment by mdahamiwal** [comment]
I was referring to the pattern implemented here https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/Tooltip . You attach `aria-describedby` on the target element and pass the same `id` to the tooltip host?

**Comment by cschleiden** [comment]
Being investigated in #1628
